localnet/poktrolld/config/relayminer_config_full_example.yaml

```
@@ -0,0 +1,108 @@
+proxies:
+    # Name of the proxy. It will be used to reference in a supplier. Must be unique.
+    # Required.
+    # TODO_CONSIDERATION: if we enforce DNS compliant names, it can potentially
+    # become handy in the future.
+    # More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names
+  - name: http-example
+    # Type of proxy: currently only http is supported but will support more
+    # (https, tcp, quic ...) in the future.
+    # MUST match the type of the supplier.
+    # Required.
+    type: http
+    # Hostname to open port on. Use 0.0.0.0 in containerized environments,
+    # 127.0.0.1 with a reverse-proxy when there's another process on localhost
+    # that can be used as a reverse proxy (nginx, apache, traefik, etc.).
+    # Required.
+    host: 127.0.0.1:8080
+
+  # TODO_IMPROVE: https is not currently supported, but this is how it could potentially look.
+  # - name: example-how-we-can-support-https
+  #   type: https
+  #   host: 0.0.0.0:8443
+  #   tls:
+  #     enabled: true
+  #     certificate: /path/to/crt
+  #     key: /path/to/key
+
+# Suppliers are different services that can be offered through RelayMiner.
+# When a supplier is configured to use a proxy and staked appropriately,
+# the relays will start flowing through RelayMiner.
+suppliers:
+    # Name of the supplier offered to the network.
+    # Must be unique.
+    # Required.
+  - name: ethereum
+    # Type of how the supplier offers service through the network.
+    # Must match the type of the proxy the supplier is connected to.
+    # Required.
+    type: http
+    # Configuration of the service offered through RelayMiner.
+    service_config:
+      # URL RelayMiner proxies the requests to.
+      # Required.
+      url: http://anvil.servicer:8545
+      # Authentication for the service.
+      # HTTP Basic Auth: https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication
+      # Optional.
+      authentication:
+        username: user
+        password: pwd
+
+      # TODO_IMPROVE: This is not supported in code yet,
+      # but some services authenticate via a header.
+      # Example, if the service requires a header like `Authorization: Bearer <PASSWORD>`
+      # Authorization: Bearer <PASSWORD>
+      # Optional.
+      headers: {}
+
+    # A list of hosts the HTTP service is offering.
+    # When linked to the proxy, that hostname is going to be used to route the
+    # request to the correct supplier.
+    # That hostname is what the user should stake the supplier for.
+    # Must be unique within a proxy/proxies it is set up on;
+    # in other words, one proxy can't offer the same hostname more than once.
+    # The `name` of the supplier is automatically added to the hosts section
+    # for potential troubleshooting/debugging purposes
+    # Required.
+    hosts:
+      - ethereum.devnet1.poktroll.com
+      # - ethereum # <- this part is be added automatically.
```

Can you explain WHY? for what purposes / use case?
```suggestion
      # - ethereum # <- this part is added automatically.
```